config/application.rb

```diff
@@ -44,10 +44,6 @@ class Application < Rails::Application
 
     config.mailer = config_for(:mailer)
 
-    # Make it Rails 7 ready
-    config.autoload_paths += %W{#{Rails.root}/app}
-    config.eager_load_paths += %W{#{Rails.root}/app}
```

Comment:
because it is no longer needed as far as I can see. In fact it is behaviour I believe is removed in Rails 8. I will check it on UAT from branch as sometimes eager loading issues don't appear until then.